feat(changelog): key-manager-added-new-post-quantum-asymmetric-signature 2026-06-25 (#6635)
scaleway/docs-content: changelog/june2026/2026-06-25-key-manager-added-new-post-quantum-asymmetric-signature.mdx

---

> title: New Post Quantum Asymmetric Signature Algorithms

Reviewer: ```suggestion
title: New post-quantum asymmetric signature algorithms
```

status: added
date: 2026-06-25
category: security-identity
product: key-manager
---

> Key Manager is now supporting three new algorithms for Asymmetric Signature : ML_DSA_44, ML_DSA_65 and ML_DSA_87.

Reviewer: ```suggestion
Key Manager now supports three new algorithms for asymmetric signature: ML_DSA_44, ML_DSA_65, and ML_DSA_87.
```

> They are all NIST-standardized post quantum signature algorithm.

Reviewer: ```suggestion
They are all NIST-standardized post-quantum signature algorithms.
```
